scm-ui/ui-webapp/src/repos/permissions/containers/CreatePermissionForm.tsx

```diff
@@ -193,22 +196,22 @@ const CreatePermissionForm: FC<Props> = ({
               <GroupAutocomplete
                 autocompleteLink={links.groups.href}
                 valueSelected={selectName}
-                value={permission.value || ""}
+                value={permission.value || empty}
```

Comment:
Cannot it just be `permission.value` ? The Autocomplete takes `SelectValue | undefined` which should just work. Or is this a different use-case ?